app/controllers/tournaments_controller.rb

```diff
@@ -20,6 +42,18 @@ def scores
   end
 
   def success
+    unless Tournament.exists?(params[:id])
+      render json: { error: "Tournament Not Found" }, status: :not_found
+    else
+      begin
+        results = Result.select(:answers).where(tournament_id: params[:id])
+        statistics = StatisticsCalculator.success_per_question(results)
+        render json: statistics, status: :ok 
+      rescue Exception => e
```

Comment:
```suggestion
      rescue StandardError => e
```
Prefer rescuing StandardError to Exception: https://www.honeybadger.io/blog/ruby-exception-vs-standarderror-whats-the-difference/

Reply:
Also, you won't need this when you move the rescue to ApplicationController